### PR TITLE
Random fixes from RN 0.59 branch

### PR DESCRIPTION
**Pull request by chrisnojima**
while making rn 59 i came across these fixes

https://github.com/keybase/client/pull/16561

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,7 +41,6 @@ jobs:
         working_directory: shared
         environment:
           REACT_NATIVE_MAX_WORKERS: 1
-          BABEL_PLATFORM: ReactNative
```
Comment (chrisnojima) [inline .circleci/config.yml]: found a way to not need this!

File: shared/constants/chat2/message.js [hunk 2]
```diff
@@ -356,7 +356,7 @@ const makeMessageSystemChangeRetention: I.RecordFactory<MessageTypes._MessageSys
     isInherit: false,
     isTeam: false,
     membersType: 0,
-    policy: RPCChatTypes.RetentionPolicy,
```
Comment (chrisnojima) [inline shared/constants/chat2/message.js]: this isn't a value so it was null!

**Comment by chrisnojima** [comment]
@keybase/react-hackers

File: shared/constants/chat2/message.js [hunk 3]
```diff
@@ -356,7 +356,7 @@ const makeMessageSystemChangeRetention: I.RecordFactory<MessageTypes._MessageSys
...
+    policy: RPCChatTypes.commonRetentionPolicyType.none,
```
Comment (buoyad) [inline shared/constants/chat2/message.js]: This is typed as
```
export type RetentionPolicy = {typ: 1, retain: ?RpRetain} | {typ: 2, expire: ?RpExpire} | {typ: 3, inherit: ?RpInherit} | {typ: 4, ephemeral: ?RpEphemeral}
```
it's probably better to make this `{typ: RPCChatTypes.commonRetentionPolicyType.none}`. I think this should be changed to the UI type but that doesn't need to be done here.

Comment (buoyad) [reply shared/constants/chat2/message.js]: `canChange` in this object actually isn't in `_MessageSystemChangeRetention` either. TIL flow doesn't check these.

File: .circleci/config.yml (hunk 1 above)
Comment (Ikpeogu) [reply .circleci/config.yml:44]: 359

Comment (Ikpeogu) [reply .circleci/config.yml:44]: How it works

Comment (Ikpeogu) [reply .circleci/config.yml:44]: Okay nice

**Review by Ikpeogu — COMMENTED** [review]
Wow

**Review by Ikpeogu — COMMENTED** [review]
It
Duplicate of #

File: shared/constants/chat2/message.js (hunk 2 above)
Comment (Ikpeogu) [reply shared/constants/chat2/message.js:359]: https://user-images.githubusercontent.com/86695676/123932968-da3b5300-d989-11eb-8b55-eef21c1c964e.mp4

Comment (Ikpeogu) [reply shared/constants/chat2/message.js:359]: Is that good

**Pull request merged**